src/runners/rfq_maker_balance_cache_runner.ts

```diff
@@ -141,7 +166,13 @@ export async function cacheRfqBalancesAsync(
     const balancesCallInput = splitValues(makerTokens);
 
     const updateTime = new Date();
-    const erc20Balances = await getErc20BalancesAsync(balanceCheckerContractInterface, balancesCallInput, codeOverride);
+    let erc20Balances: string[];
+    try {
+        erc20Balances = await getErc20BalancesAsync(balanceCheckerContractInterface, balancesCallInput, codeOverride);
+    } catch (err) {
+        requestErrors += 1;
```

Comment:
You are exporting this function, which means that it's usage can happen even outside of this file/module. However, this function does change state witin the module, which isn't the best design imo. Can you update the `requestErrors` outside of `cacheRfqBalancesAsync` instead?

Reply:
abstracted this away